### DISCUSSION TITLE
Improve documentation for `options`: `waitForTimeout` and `waitForSelector`

### DISCUSSION THREAD

**Discussion by oscard0m** [percy/percy-storybook#560] [General]
## The problem
I would like to improve `Percy Docs` regarding `storybook` options.

## Environment

not relevant

## Details

I'm trying to use `waitForTimeout` and `waitForSelector` and it's not clear the kind of values these parameters are expecting:
- is the `timeout` expecting milliseconds or another time unit (by logic I guess is milliseconds)
- Does it support integers or does it support strings as well? (spoiler: after failing I noticed it only supports integers)
- What about the `waitForSelector`? Does it support "complex" DOM selectors (by `data-attribute` for example), or only some basic selectors by `id` and/or `className`? (by logic I guess it supports any valid selector you can pass to `document.querySelector()`). Also, is it possible to waitFor a certain number of DOMElements which respond to the same selector?

### Context
https://github.com/percy/percy-storybook/pull/558

**Comment by wwilsman** [comment]
We definitely need to improve our docs, and your assumptions are correct. For posterity:

- `waitForTimeout` is passed as the second arg to `setTimeout(resolve, timeout)`
- `waitForSelector` is passed directly to `document.querySelector(selector)`

These options are accepted by more than just this SDK as well. It's on our radar to update our docs in the near future. 👍

**Reply by oscard0m** [reply]
Thanks for clarifying this so quickly. If you want to open a PR for upgrading it let me know but sounds like you will work on a big update in your docs.
